UPGRADE.md

```diff
@@ -0,0 +1,115 @@
+# Upgrade
+
+## 3.0
+
+### Switch from LiipThemeBundle to SyliusThemeBundle
```

Comment:
@lsmith77 Parts of this UPGRADE.md could be used in the LiipThemeBundle docs when the project will be marked as abandoned.